doc/rados/deployment/ceph-deploy-osd.rst

```diff
@@ -32,8 +32,8 @@ Zap Disks
 To zap a disk (delete its partition table) in preparation for use with Ceph,
 execute the following::
 
-	ceph-deploy disk zap {osd-server-name}:{disk-name}
-	ceph-deploy disk zap osdserver1:sdb
+	ceph-deploy disk zap {osd-server-name} {disk-name}
+	ceph-deploy disk zap osdserver1 /dev/sdb /dev/sdc
```

Comment:
are you sure about this? as i found

https://github.com/ceph/ceph/blob/f61d002825fdaad11ac373855f25d90ab9bedf13/qa/tasks/ceph_deploy.py#L245

Reply:
that qa task is probably not accurate...

```
ceph-deploy disk zap node3:sdy     
[ceph_deploy.conf][DEBUG ] found configuration file at: /home/alfredo/.cephdeploy.conf
[ceph_deploy.cli][INFO  ] Invoked (2.0.2): /home/alfredo/.virtualenvs/ceph-deploy/bin/ceph-deploy disk zap node3:sdy
[ceph_deploy.cli][INFO  ] ceph-deploy options:
[ceph_deploy.cli][INFO  ]  verbose                       : False
[ceph_deploy.cli][INFO  ]  quiet                         : False
[ceph_deploy.cli][INFO  ]  username                      : None
[ceph_deploy.cli][INFO  ]  overwrite_conf                : False
[ceph_deploy.cli][INFO  ]  ceph_conf                     : None
[ceph_deploy.cli][INFO  ]  cluster                       : ceph
[ceph_deploy.cli][INFO  ]  subcommand                    : zap
[ceph_deploy.cli][INFO  ]  cd_conf                       : <ceph_deploy.conf.cephdeploy.Conf object at 0x7f699a782240>
[ceph_deploy.cli][INFO  ]  default_release               : False
[ceph_deploy.cli][INFO  ]  func                          : <function disk at 0x7f699a82da60>
[ceph_deploy.cli][INFO  ]  host                          : None
[ceph_deploy.cli][INFO  ]  disk                          : ['node3:sdy']
[ceph_deploy.cli][INFO  ]  debug                         : False
Traceback (most recent call last):
  File "/home/alfredo/.virtualenvs/ceph-deploy/bin/ceph-deploy", line 11, in <module>
    load_entry_point('ceph-deploy', 'console_scripts', 'ceph-deploy')()
  File "/home/alfredo/python/ceph-deploy/ceph_deploy/cli.py", line 170, in main
    _main(args=args, namespace=namespace)
  File "/home/alfredo/python/ceph-deploy/ceph_deploy/util/decorators.py", line 68, in newfunc
    return f(*a, **kw)
  File "/home/alfredo/python/ceph-deploy/ceph_deploy/cli.py", line 166, in _main
    return args.func(args)
  File "/home/alfredo/python/ceph-deploy/ceph_deploy/osd.py", line 438, in disk
    disk_zap(args)
  File "/home/alfredo/python/ceph-deploy/ceph_deploy/osd.py", line 319, in disk_zap
    raise RuntimeError('zap command needs both HOSTNAME and DISK but got "%s %s"' % (hostname, disk))
RuntimeError: zap command needs both HOSTNAME and DISK but got "None node3:sdy"
```

Using the proposed change is correct (to me):

```
ceph-deploy disk zap node3 /dev/sdy
[ceph_deploy.conf][DEBUG ] found configuration file at: /home/alfredo/.cephdeploy.conf
[ceph_deploy.cli][INFO  ] Invoked (2.0.2): /home/alfredo/.virtualenvs/ceph-deploy/bin/ceph-deploy disk zap node3 /dev/sdy
[ceph_deploy.cli][INFO  ] ceph-deploy options:
[ceph_deploy.cli][INFO  ]  verbose                       : False
[ceph_deploy.cli][INFO  ]  quiet                         : False
[ceph_deploy.cli][INFO  ]  username                      : None
[ceph_deploy.cli][INFO  ]  overwrite_conf                : False
[ceph_deploy.cli][INFO  ]  ceph_conf                     : None
[ceph_deploy.cli][INFO  ]  cluster                       : ceph
[ceph_deploy.cli][INFO  ]  subcommand                    : zap
[ceph_deploy.cli][INFO  ]  cd_conf                       : <ceph_deploy.conf.cephdeploy.Conf object at 0x7f21d3866278>
[ceph_deploy.cli][INFO  ]  default_release               : False
[ceph_deploy.cli][INFO  ]  func                          : <function disk at 0x7f21d3911a60>
[ceph_deploy.cli][INFO  ]  host                          : node3
[ceph_deploy.cli][INFO  ]  disk                          : ['/dev/sdy']
[ceph_deploy.cli][INFO  ]  debug                         : False
[ceph_deploy.osd][DEBUG ] zapping /dev/sdy on node3
[node3][DEBUG ] connection detected need for sudo
[node3][DEBUG ] connected to host: node3 
[ceph_deploy.osd][INFO  ] Distro info: CentOS Linux 7.6.1810 Core
[node3][INFO  ] Running command: sudo /usr/sbin/ceph-volume lvm zap /dev/sdy
[node3][WARNIN] --> Zapping: /dev/sdy
[node3][WARNIN] --> --destroy was not specified, but zapping a whole device will remove the partition table
[node3][WARNIN] Running command: /usr/sbin/wipefs --all /dev/sdy
[node3][WARNIN] Running command: /bin/dd if=/dev/zero of=/dev/sdy bs=1M count=10
[node3][WARNIN] --> Zapping successful for: <Raw Device: /dev/sdy>
```